test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -79,18 +83,26 @@ def clone(x):
 
         return tuple(clone(x) for x in inputs)
 
-    def _generate_reference_quantized_model(self, mod, inputs):
-        export_model = capture_pre_autograd_graph(
-            mod,
-            inputs,
-        )
-        quantizer = X86InductorQuantizer()
-        quantizer.set_global(xiq.get_default_x86_inductor_quantization_config())
-        prepare_model = prepare_pt2e(export_model, quantizer)
-        prepare_model(*inputs)
-        convert_model = convert_pt2e(prepare_model)
-        torch.ao.quantization.move_exported_model_to_eval(convert_model)
-        return convert_model
+    def _generate_reference_quantized_model(self, mod, inputs, is_qat=False):
+        maybe_no_grad = contextlib.nullcontext() if is_qat else torch.no_grad()
+        with maybe_no_grad:
+            export_model = capture_pre_autograd_graph(
+                mod,
+                inputs,
+            )
+            quantizer = X86InductorQuantizer()
+            quantizer.set_global(
+                xiq.get_default_x86_inductor_quantization_config(is_qat=is_qat)
+            )
+            prepare_model = (
+                prepare_qat_pt2e(export_model, quantizer)
+                if is_qat
+                else prepare_pt2e(export_model, quantizer)
+            )
+            prepare_model(*inputs)
+            convert_model = convert_pt2e(prepare_model)
```

Comment:
please add fold_quantize=True here, we are thinking of making this default soon

Reply:
Thanks for the comment and add `fold_quantize=True` into `convert_pt2e`.